backend/script.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const axios = require('axios');
+const cron = require('node-cron');
+const MyModel = require('./models/Brand');
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+const DB_URI = 'mongodb+srv://Admin:Admin69@cluster0.bs2zgrh.mongodb.net/test';

```

Comment:
`DB_URI` contains your mongodb atlas connection details and user credentials, you should not keep them in code here, you should put link in `.env` and use it here.